src/lib/AnnotationControls.ts

```diff
@@ -84,21 +84,53 @@ export default class AnnotationControls {
     private handleFullscreenExit = (): void => this.handleFullscreenChange(false);
 
     /**
-     * Region comment button click handler
+     * Deactive current control button
      */
-    private handleRegionClick = (onRegionClick: RegionHandler) => (event: MouseEvent): void => {
+    public deactivateCurrentControl = (): void => {
+        if (!this.currentActiveControl) {
+            return;
+        }
+
+        switch (this.currentActiveControl) {
+            case ANNOTATION_MODE.region:
+                this.currentActiveControl = null;
```

Comment:
It looks like this happens no matter what. Should it be moved outside of the switch?

Reply:
> It looks like this happens no matter what. Should it be moved outside of the switch?

Yeah, but I need to create another variable to store the original value of it, because the switch is using the its value.